src/commands.ts

```diff
@@ -260,6 +267,33 @@ const slashCommands: SlashCommandsType = {
         },
         text: 'Show details of when each tracked player was last updated',
         privileged: true
+    },
+    role: {
+        options: [{
+            type: ApplicationCommandOptionType.Mentionable,
+            name: 'role',
+            description: 'Server Role',
+            required: true
+        }],
+        execute: async (interaction) => {
+            const guild = getInteractionGuild(interaction);
+            const mentionable = interaction.options.getMentionable('role', true);
+            if (mentionable instanceof Role) {
+                await interaction.deferReply({ ephemeral: true });
+                const newPrivilegedRole = mentionable;
+                await pgStorageClient.writePrivilegedRole(guild.id, newPrivilegedRole.id);
+                state.setPrivilegedRole(guild.id, newPrivilegedRole);
+                await guildCommandRolePermissionsManager.update(guild);
+                await interaction.editReply(`${newPrivilegedRole} can now use **/track**, **/remove**, **/clear**, and **/channel**`);
+            } else {
+                await interaction.reply({
+                    content: 'Can only give a mentionable role (**@everyone**, **@Gamers**, etc.) privileges',
+                    ephemeral: true
+                });
+            }
+        },
+        text: 'Set a non-admin server role that can use /track, /remove, /clear, and /channel',
+        privileged: true
```

Comment:
How are we guaranteeing that only admins can use `/role`? If this is just `privileged`, then that means anyone with the privileged role can then themselves set the privileged role

Reply:
Is that what the logic with the TODO does?

Reply:
No, because /role is not a `guild` command.